code/game/machinery/doors/airlock.dm

```diff
@@ -598,7 +598,7 @@
 			. += "There's a [note.name] pinned to the front..."
 			. += note.examine(user)
 	if(seal)
-		. += "It's been braced with a pneumatic seal."
+		. += "It's been braced with a pneumatic airlock seal."
```

Comment:
```suggestion
		. += "It's been braced with \a [seal]."
```

`seal` is an item. Double-check this one for me.

Reply:
It works and now we can have upgraded seals (please don't)